Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
         .library(name: "SocketIO", targets: ["SocketIO"])
     ],
     dependencies: [
-        .package(url: "https://github.com/daltoniam/Starscream", .exactItem("4.0.6")),
+        .package(url: "https://github.com/daltoniam/Starscream", .exactItem("4.0.8")),
```

Comment:
I think this should be `upToNextMajor`.

Reply:
I also believe that's the way to go. However, since I don't fully understand the intentions behind the changes made in https://github.com/socketio/socket.io-client-swift/pull/1451/commits/3c81d0ee55111a80a8f39b9c61782887ec98ceb4 , I feel it's not appropriate for me to do it.